lib/eventFactory.js

```diff
@@ -581,16 +581,26 @@ async function getJobParameters(config) {
     const jobs = await pipeline.getJobs();
 
     jobs.forEach(job => {
-        // Ignore PR jobs (Ex: PR-2:component) as the parameters would be included from the parent jobs (Ex: component)
-        if (job.prParentJobId === null || job.prParentJobId === undefined) {
-            const jobParameters = job.permutations[0].parameters; // TODO: Revisit while supporting matrix job
+        const jobParameters = job.permutations[0].parameters; // TODO: Revisit while supporting matrix job
 
-            if (jobParameters) {
-                allowedParameters[job.name] = validateAndMergeParameters(
+        if (eventConfig.prNum) {
+            // For PR events, include only the parameters of the PR job for that PR
+            const prJobPrefix = `PR-${eventConfig.prNum}:`;
```

Comment:
You can use [`job.isPR()`](https://github.com/screwdriver-cd/models/blob/2213ac9b1b3976c5449fe35271ad155068965872/lib/job.js#L130-L132) method.

Reply:
Thanks. I used [`job.prNum`](https://github.com/screwdriver-cd/models/blob/2213ac9b1b3976c5449fe35271ad155068965872/lib/job.js#L148-L150) to make the condition concise.